debian/control

```diff
@@ -10,8 +10,9 @@ Homepage: <insert the upstream URL, if relevant>
 
 Package: rpi-chromium-mods-kano
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, chromium-browser
-Replaces: chromium-browser
+Depends: ${shlibs:Depends}, ${misc:Depends}, chromium-browser (>= 55) [armhf], chromium (>= 55) [!armhf]
+Replaces: chromium-browser, rpi-chromium-mods
```

Comment:
I know that we had `Replaces: chromium-browser` before but is this right given that it also depends on it? It's a bit weird.

Reply:
As discussed, Replaces is correct for chromium-browser but it should be 'conflicts' for rpi-chromium-mods. I've made that change.